common/src/main/java/org/jboss/gm/common/Configuration.java

```diff
@@ -48,6 +48,9 @@ default Properties getProperties() {
     @Key("AProxDeployUrl")
     String deployUrl();
 
+    @Key("deployPlugin")
+    String deployPlugin();
```

Comment:
From the look of the usage deployPlugin only has two valid values? So why not make it a enum for improved static typing?

Reply:
( Discussed offline ; we'll leave this for now ).